src/gitsh.c

```diff
@@ -1,12 +1,25 @@
 #include <err.h>
+#include <ruby.h>
 #include <sysexits.h>
-#include <unistd.h>
 
 int
 main(int argc, char *argv[])
 {
-    if (execv(GITSH_RB_PATH, argv) == -1) {
-        err(EX_SOFTWARE, GITSH_RB_PATH);
+    int ruby_argc;
+    char *ruby_argv[];
```

Comment:
I get a compile error on this line when I try to build this locally:

```
src/gitsh.c:9:11: error: definition of variable with array type needs an
      explicit size or an initializer
    char *ruby_argv[];
          ^
1 error generated.
```

This version built successfully:

```suggestion
    char **ruby_argv;
```

Reply:
Yeah, I ran into this myself 👍 pushing untested code…

Reply:
I'll push a commit to fix this, to see what happens on CI